### PR TITLE
temp: disable cancun unit tests with identified root causes pending m…

### PR DESCRIPTION
**Pull request by amkCha**
…erge

I am disabling Cancun unit tests that have identified root causes that have pending PRs to be merged to have Cancun pass and identify new tests that are breaking with new PR more easily

Root causes are mainly 
- the selfdestruct bug flagged in an issue, pending PR https://github.com/Consensys/linea-tracer/pull/2184
- a comparison method fix in InstructionByteStringPrefix https://github.com/Consensys/linea-tracer/pull/2189/files
- merging KZG PR to have precomp section added in list + isAddressWarm PR https://github.com/Consensys/linea-tracer/pull/2152
- look up edge case, either change spec or tracer https://github.com/Consensys/linea-specification/issues/233
- Advanced create test 1 transaction, issue opened https://github.com/Consensys/linea-tracer/issues/2190
- constraint PR for resA and resB in mxp https://github.com/Consensys/linea-constraints/pull/730

### CODE REVIEW

File: testing/src/main/java/net/consensys/linea/reporting/TracerTestBase.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class TracerTestBase {
   @BeforeEach
   public void init(TestInfo testInfo) {
       TracerTestBase.testInfo.chainConfig =
-              switch (getForkOrDefault("LONDON")) {
+              switch (getForkOrDefault("CANCUN")) {
```
Comment (DavePearce) [inline testing/src/main/java/net/consensys/linea/reporting/TracerTestBase.java]: Can you change this back to `LONDON` ?

**Review by DavePearce — APPROVED** [review]
Looks good other than `TracerTestBase`
